.github/workflows/sync.yml

```diff
@@ -28,12 +28,18 @@ jobs:
           npm upgrade pogo-data-generator
           npm upgrade pogo-protos
           npm start
-      - name: Configure git
+      - name: Check for changes
+        run: |
+          if git diff --quiet; then
+            echo "changes_exist=false" >> $GITHUB_ENV
```

Comment:
and yes `git diff --quiet` returns 0 if unchanged. and in bash 0 means true, confusing but :shrug: